torchtune/models/phi3/_position_embeddings.py

```diff
@@ -40,6 +40,9 @@ def __init__(
         self.max_seq_len = max_seq_len
         self._rope_init()
 
+    def reset_parameters(self):
```

Comment:
Do we need this? I don't think we're using this anymore since we don't init on meta device.